src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -5438,6 +5440,13 @@ public override IEnumerable<IOperation> Children
                 {
                     yield return Initializer;
                 }
+                if (!IgnoredArguments.IsEmpty)
```

Comment:
>if (!IgnoredArguments.IsEmpty) [](start = 16, length = 30)

Please add a targeted test for the Children property.

Reply:
Done.

---
In reply to: [149217374](https://github.com/dotnet/roslyn/pull/23008#discussion_r149217374) [](ancestors = 149217374)